src/text_file_functions.cpp

```diff
@@ -326,15 +325,23 @@ pos_size_type AOApplication::get_element_dimensions(QString p_identifier,
 QString AOApplication::get_design_element(QString p_identifier, QString p_file,
                                           QString p_char)
 {
+  QString design_char_ini_path =
+      get_theme_path("misc/" + get_chat(p_char) + "/" + p_file);
   QString char_ini_path =
       get_base_path() + "misc/" + get_chat(p_char) + "/" + p_file;
   QString design_ini_path = get_theme_path(p_file);
   QString default_path = get_default_theme_path(p_file);
-  QString f_result = read_design_ini(p_identifier, char_ini_path);
+  QString f_result = read_design_ini(p_identifier, design_char_ini_path);
   if (f_result == "") {
-    f_result = read_design_ini(p_identifier, design_ini_path);
-    if (f_result == "")
-      f_result = read_design_ini(p_identifier, default_path);
+    f_result = read_design_ini(p_identifier, char_ini_path);
+    if (f_result == "") {
+      f_result = read_design_ini(p_identifier, design_ini_path);
+      if (f_result == "") {
+        f_result = read_design_ini(p_identifier, default_path);
+        if (f_result == "")
+          return "";
+      }
+    }
```

Comment:
This is where an array comes in very handy:
```cpp
QStringList paths {
  get_theme_path("misc/" + get_chat(p_char) + "/" + p_file),
  get_base_path() + "misc/" + get_chat(p_char) + "/" + p_file,
  get_theme_path(p_file),
  get_default_theme_path(p_file)
};
for (const QString &path : paths) {
  QString value = read_design_ini(p_identifier, path);
  if (!value.isEmpty())
    return value;
}
return "";
```

Reply:
Thanks! Resolved as of [`eb988d9`](https://github.com/AttorneyOnline/AO2-Client/pull/302/commits/eb988d920942ed1bcf222d6827f230d3aaa654e7)